### DISCUSSION TITLE
Dolby Atmos (e-ac-3) support?

### DISCUSSION THREAD

**Discussion by sclsj** [mikebrady/shairport-sync#1610] [Q&A] [answered]
I've read somewhere that AirPlay 2 supports Dolby Atmos streaming. Is that true? If so, when playing Apple Music with Dolby Atmos turned on, will shairport receive Dolby Atmos audio without any additional configuration?

**Comment by mikebrady** [comment, marked as answer]
I don't think so, I'm afraid. It seems that what is forwarded to Shairport Sync is regular stereo.

**Reply by sclsj** [reply]
Just want to confirm you did try with with a Dolby Atmos track and verified the track shows Dolby Atmos icon during playback?

**Reply by mikebrady** [reply]
Have played Dolby Atmos tracks alright, but they are decoded as regular stereo AAC.

**Comment by Hk1020** [comment]
I’ve just noticed that if I play to my homepod minis only they play in Dolby Atmos but if I play to my raspberry pi instead it is regular stereo lossless. But maybe the ipad isn’t actually playing via Airplay to the homepod as the homepods can play Apple Music without an airplay device? - I should also note that I am not using the latest shairport. I am at 4.1-dev-683-g06c69c69-AirPlay2-OpenSSL-Avahi-ALSA-pa-soxr-metadata-mqtt-dbus-sysconfdir:/etc

**Reply by mikebrady** [reply]
Yeah, I've noticed this change in the label from Atmos to Lossless alright, but the feed to Shairport Sync remains regular stereo AAC, I'm afraid. Dunno what's going to the HomePod though...

**Comment by Hk1020** [comment]
> On Jan 31, 2024, at 15:25, Mike Brady ***@***.***> wrote:
> 
> Yeah, I've noticed this change in the label from Atmos to Lossless alright, but the feed to Shairport Sync remains regular stereo AAC, I'm afraid.
> 
It certainly sounds like regular stereo. 
> Dunno what's going to the HomePod though...
> 
Would be curious to know if Airplay actually supports Atmos by now. Is there any news?

**Reply by mikebrady** [reply]
I'd be very surprised indeed if Apple wasn't working on it...

**Reply by Krushna-Shah** [reply]
with ios18, airplay support for dolby atmos from ‘iphone to homepod, and select third-party, Airplay-compatible audio equipment’. Please, Please, Please implement this, wireless atmos like this would enable so many cool atmos systems that are wireless
